doc/appendix/release-history/v0.6.rst

```diff
@@ -0,0 +1,263 @@
+.. Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _release_v0.6.0:
+
+Version 0.6.0
+-------------
+*Released 2025/04/23*
+
+Version 0.6.0 is a major update to Celeritas featuring:
+
+- High-level integration utilities for Geant4 applications
+- Optical physics implementation including volumetric physics
+- Advanced field mapping capabilities with 3D cylindrical and cartesian field maps and volume-specific uniform fields
+- Extended geometry conversion and optimization-oriented features for ORANGE
+- Support for VecGeom 2.0
+- Spline interpolation for cross-section calculations and energy loss
+
+A few minor features are noteworthy:
+
+- Improved debugging utilities for diagnosing stuck/errored tracks
+- Support for sensitive detectors in replica/parameterized volumes
+- Better handling of tracks that fail or get stuck during simulation
+
+Notable bug fixes include:
+
+- Fixed navigation and mapping of reflecting volumes for VecGeom
+- Resolved crashes related to process construction and memory access
+- Fixed issues with GDML file loading and SD hit scoring
+- Fixed Urban MSC scattering distribution (backported to v0.5.3)
+
+Some interfaces have been removed:
+
+- Obsolete demonstration apps and celer-dump-data
+- Macro setup from celer-g4
+- The ability to run on the default stream
+
+Known deficiencies:
+
+- Surface optical physics (reflection, refraction) is not yet implemented
+- ORANGE performance is significantly degraded compared to v0.5
+
+Changes since v0.5.0, excluding those released in v0.5.3, follow.
+
+New features
+------------
+
+* Add optical "locate vacancies" and pre-step actions *(@amandalund, #1441)*
+* Extend debug utilities for stuck/errored tracks *(@sethrj, #1451)*
+* Add CELER_NONFATAL_FLUSH to cleanly kill stuck tracks *(@sethrj, #1455)*
+* Add optical absorption model with imported data *(@hhollenb, #1440)*
+* Add multilevel "volume instance"  *(@sethrj, #1461)*
+* Reconstruct Geant4 navigation history from stack *(@sethrj, #1466)*
+* Add "cut" for individual track steps *(@sethrj, #1467)*
+* Improve error checking and diagnostic output for managing GPU *(@sethrj, #1464)*
+* Add distance-to-bbox function *(@elliottbiondo, #1446)*
+* Add multi-level state to detector step result *(@sethrj, #1471)*
+* Override and save Geant4 dataset variables at configure time *(@pcanal, #1475)*
+* Implement spline interpolation for cross-section calculation *(@lebuller, #1444)*
+* Implement optical Rayleigh model *(@hhollenb, #1469)*
+* Allow core and optical loops to have different numbers of track slots *(@amandalund, #1491)*
+* Add muon decay interactor *(@stognini, #1456)*
+* Add option to use spline-interpolated energy loss *(@lebuller, #1496)*
+* Add option to use spline-interpolated energy loss *(@lebuller, #1496)*
```

Comment:
Duplicate

Reply:
Yeah that was a weird GitHub glitch. It showed up in the history too